src/web/docusaurus/docs/contributing/database-maintenance.md

```diff
@@ -0,0 +1,55 @@
+---
+sidebar_position: 8
+---
+
+# Database Maintenance
+
+The follow describes how to manually work with the Postgres database.
+
+## Backup (Manual)
+
+1. Go to Portainer ([staging][portainer-staging] or [production][portainer-production]) and locate the `pg-backup-cron-job` container
+2. Click the **Console** button and select `bin/sh` in order to get a shell in the container
+3. Run a backup:
+
+```sh
+/etc/periodic/daily/create-backup
+```
+
+This will create a new dump file in `/var/opt/pg_dumps` with the current date and time.
+
+## Restore a Backup
+
+1. Go to Portainer ([staging][portainer-staging] or [production][portainer-production]) and locate the `pg-backup-cron-job` container
+1. Click the **Console** button and select `bin/sh` in order to get a shell in the container
+1. Run a restore:
+
+```sh
+ls /var/opt/pg_dumps
+# Locate your desired dump file <absolute-path-to-dump-file>
+$ ./restore-backup <absolute-path-to-dump-file>
+```
+
+This will attempt to restore the database to the state in the dump file. NOTE: some operations will fail if tables exist or if the schema in the dump file does not match with the schema in the database, or if Postgres can't make it work for some reason.
+
+## Migrations
+
+1. Go to Portainer ([staging][portainer-staging] or [production][portainer-production])
+2. Click on the `primary` environment, which has all of our containers.
+3. Click **Containers**
+4. Get the current postgres password going to the `supabase-db` container, and inspecting the `POSTGRES_PASSWORD` environment variable. Copy this (you will need it below).
+5. Go back to **Containers** and click **Add Container** to add a new container
+6. Enter a **Name**: `db-migrations`
+7. Select the `docker.cdot.systems` **Registry**. NOTE: if you don't see it, add it via the **Registries** navigation menu on the left.
```

Comment:
Same thing here. However, I can imagine how it looks if we are listing by number, and the **Note** box comes in, this could be considered exception?